src/state/users/selectors.js

```diff
@@ -3,6 +3,7 @@ import { createSelector } from 'reselect';
 export const getUsers = state => state.users;
 export const getUsersIdList = state => state.users.list;
 export const getUsersMap = state => state.users.map;
+export const getUser = state => state.users.selected;
```

Comment:
Please rename as getSelectedUser, as getUser could be intended as returning the currently logged user

Reply:
Is not logged user, but the selected user that you want get the information